pkg/virt-launcher/virtwrap/device/hostdevice/sriov/hostdev_test.go

```diff
@@ -164,6 +164,54 @@ var _ = Describe("SRIOV HostDevice", func() {
 			Expect(devices, err).To(Equal([]api.HostDevice{expectHostDevice1}))
 		})
 
+		DescribeTable("create two devices with custom guest PCI address",
+			func(iface1HasCustomGuestPCIAddress, iface2HasCustomGuestPCIAddress bool) {
```

Comment:
We do not like booleans as inputs. I think that you can pass in PCI Addresses directly (nil in case it is not used. You should not need to ask a lot of questions in the common code.

Reply:
It was changed to strings because it was more convenient to pass the PCI addresses as strings and convert then to the PCI address structs.